.github/workflows/size-diff.yml

```diff
@@ -0,0 +1,55 @@
+# Calculates the size diffs for the each example binary. Runs when a pull
+# request is created or modified.
+
+name: size-diff
+on: pull_request
+
+jobs:
+  size-diff:
+    # Using ubuntu-latest can cause breakage when ubuntu-latest is updated to
+    # point at a new Ubuntu version. Instead, explicitly specify the version, so
+    # we can update when we need to. This *could* break if we don't update it
+    # until support for 18.04 is dropped, but it is likely we'll have a reason
+    # to update to a newer Ubuntu before then anyway.
+    runs-on: ubuntu-18.04
```

Comment:
I'd prefer 20.04, but it's not a hard requirement.

Reply:
Github lists 20.04 as "preview only". It's not very clear to me what that means. I've been interpreting "preview only" to mean that using it is discouraged.

Reply:
Ah ok. This is fine then.